### ISSUE TITLE
Spanish support

### ISSUE THREAD

**Issue by checor** [Skycoder42/QtAutoUpdater#9]
Hello!

Your library is cool, and I'm intending to implement it in one of my projects 👍 

I can provide you a spanish translation, are you interested in adding it to your repo? I can send a pull request.

Cheers!

**Comment by Skycoder42** [comment]
Of course, feel free to do so! This way it will also become available to anyone using the online repository with the next version. :+1:

**Comment by Skycoder42** [comment]
The translations have been updated on the [major branch](https://github.com/Skycoder42/QtAutoUpdater/tree/major/src/translations). It was auto-translated using https://www.matecat.com/. It would be awesome if you could verify the correctness of the generated translations. Thanks.

**Comment by checor** [comment]
Of course, seems that some messages needs grammar corrections. I will check it out in the week.
